scripts/run_pattern_check.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pre-submission script for Oppia.
+# This script checks for unaccepted text patterns in the commit.
+
+import subprocess
+
+BAD_PATTERNS = {'__author__': 'Please remove author tags from \
+this file.',
+                'datetime.datetime.now()': 'Please use datetime.datetime.u\
+tcnow() instead of datetime.datetime.now().'
+               }
+
+def _get_changed_filenames():
```

Comment:
Still needs a further change so that Travis lints all files, right?

Also, is this meant to be executed locally, or is it just for Travis? If the former, should it be combined with the pre_commit_linter.py or pre_push_hook.py scripts?